Core/Core/View/Base/RefreshProgressView.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  RefreshProgressView.swift
+//  Course
+//
+//  Created by  Stepanok Ivan on 15.04.2024.
+//
+
+import SwiftUI
+
+public struct RefreshProgressView: View {
```

Comment:
I guess it's being used for pull to refresh, how about renaming it as per functionality thoughts?